raqm.c

```diff
@@ -196,8 +203,8 @@ stack_create (int max)
 {
     Stack* stack;
     stack = (Stack*) malloc (sizeof (Stack));
-    stack->scripts = (hb_script_t*) malloc (sizeof (hb_script_t) * max);
-    stack->pair_index = (int*) malloc (sizeof (int) * max);
+    stack->scripts = (hb_script_t*) malloc (sizeof (hb_script_t) * (size_t)(max));
+    stack->pair_index = (int*) malloc (sizeof (int) * (size_t)(max));
```

Comment:
No great though have been put into choosing the types, the code just evolved “organically” as we tried things out.